### PR TITLE
Migrate CustomTabsHelper to kotlin

### PR DESCRIPTION
**Pull request by lukstbit**
## Purpose / Description

Migrates CustomTabsHelper.java to kotlin following the migration guide.

## How Has This Been Tested?

Ran the usual tests.

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/compat/customtabs/CustomTabsHelper.kt [hunk 1]
```diff
@@ -12,135 +12,129 @@
...
-    private static String sPackageNameToUse;
-
-    private CustomTabsHelper() {}
-
-    public static void addKeepAliveExtra(Context context, Intent intent) {
-        Intent keepAliveIntent = new Intent().setClassName(
-                context.getPackageName(), KeepAliveService.class.getCanonicalName());
-        intent.putExtra(EXTRA_CUSTOM_TABS_KEEP_ALIVE, keepAliveIntent);
+@KotlinCleanup("fix IDE lint issues")
+@KotlinCleanup("move uppercase const properties to the companion object")
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/compat/customtabs/CustomTabsHelper.kt:30]: Not available, as we're in an `object`

`Modifier 'companion' is not applicable inside 'object'`

**Review by david-allison — CHANGES_REQUESTED** [review]
One minor nit

File: AnkiDroid/src/main/java/com/ichi2/compat/customtabs/CustomTabsHelper.kt (hunk 1 above)
Comment (lukstbit) [reply AnkiDroid/src/main/java/com/ichi2/compat/customtabs/CustomTabsHelper.kt]: Yes:(

**Review by david-allison — APPROVED** [review]
Thanks!!!!!!!

**Pull request merged**